### DISCUSSION TITLE
Tableau Online Subscriptions with TabPy

### DISCUSSION THREAD

**Discussion by prehling** [tableau/TabPy#557] [Q&A] [answered]
I am trying to set up subscriptions for a Tableau Dashboard that utilizes TabPy. I receive the message:

"The snapshot of the view 'Operations Dashboard' could not be properly rendered."

I have tried two different types of TabPy server: Heroku and Kubernetes

I have also tried to publish one sheet with one calculated field that uses a basic python function in order to eliminate the possibility of the subscription timing out. This also returns the same error

I cannot find any documentation online of this being a known issue. 

Could somebody let me know if this is a known limitation or there is some configurations I can do on my end in order to let subscriptions work? 
![Screen Shot 2022-07-09 at 3 33 51 PM](https://user-images.githubusercontent.com/106627928/178121900-92fa6fa7-8fd6-4082-b935-79e4d46b4b65.png)

**Comment by nmannheimer** [comment, marked as answer]
Hi @prehling, this is an issue we are aware of and it has been fixed in on-premise Tableau Server. It is on our backlog to fix for Tableau Online. Thanks and I hope this helps.

**Reply by prehling** [reply]
@nmannheimer I really appreciate the response! If you don't mind me asking, do you have an estimated timeline on when that is going to be fixed? I understand if there isn't one.

**Comment by nmannheimer** [comment]
Currently we don't have a public timeline, but it is on the higher end of our priorities to fix.
